specification/resources/registry/multiregistry_create.yml

```
@@ -0,0 +1,51 @@
+operationId: multiregistry_create
```

```suggestion
operationId: registries_create
```

The operation ID is used by generated clients to name methods. So that we are consistent with godo and other resources, we probably want to use `registries` here in place of `multiregistry`.